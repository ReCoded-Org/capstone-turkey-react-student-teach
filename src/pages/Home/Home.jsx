import HomeContent from '../../components/home/HomeContent';

function Home() {
  return <HomeContent />;
}

export default Home;
