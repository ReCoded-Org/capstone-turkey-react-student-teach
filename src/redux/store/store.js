import { configureStore } from '@reduxjs/toolkit';
import addQuestionSlice from '../features/addQuestionSlice';

const store = configureStore({
  reducer: {
    addQuestionReducer: addQuestionSlice,
    signIn: userSlice,
  },
});

export default store;
