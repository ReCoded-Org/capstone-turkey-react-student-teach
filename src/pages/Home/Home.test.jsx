import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import Home from './Home';
import store from '../../redux/store/store';

it('Render when text of no HomePage found removes', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Home />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
