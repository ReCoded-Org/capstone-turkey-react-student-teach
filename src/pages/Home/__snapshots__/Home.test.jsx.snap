// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render when text of no HomePage found removes 1`] = `
<div
  className="container sm:mx-auto w-10/12  mt-8 pr-2"
>
  <div
    className="flex "
  >
    <div
      className="w-full z-20 mx-auto place-self-center w-12/12 sm:w-10/12  xl:w-7/12"
    >
      <h2
        className="text-5xl sm:text-7xl text-orange"
      >
        Student Teach
      </h2>
      <h4
        className="font-semibold text-base sm:text-2xl text-black mt-2 sm:mt-9 py-0.5"
      >
        We grow up by helping each other.
      </h4>
      <p
        className="mt-2 sm:mt-9 text-sm sm:text-xl  py-0.5 w-12/12 lg:w-10/12 "
      >
        This is a platform where you can ask your questions and help them people find answers.
      </p>
      <p
        className="mt-2 sm:mt-9 text-sm sm:text-xl  py-0.5 w-12/12 lg:w-10/12 "
      >
        There are hundreds of categories in which you can ask your questions and you can learn by traveling through them.
      </p>
    </div>
    <div
      className="hidden w-5/12  bg-cover lg:flex  bg-right w-full scale-150 -z-10"
      style={
        Object {
          "backgroundImage": "url(\\"./images/home.png\\")",
        }
      }
    >
      <button
        className="bg-orange rounded-[10px] text-xs w-44 h-10 text-[#FFFFFF] absolute inline-block bottom-[20px] xl:left-[40%] lg:left-[20%]"
      >
        Ask Question +
      </button>
    </div>
  </div>
  <button
    className=" lg:hidden mt-10 text-xs  sm:text-3xl mx-auto bg-orange rounded-[10px] sm:w-[14rem] sm:h-[4rem] w-[8rem] h-[2rem] text-[#FFFFFF] absolute inline-block  left-[30%]"
  >
    Ask Question +
  </button>
  <div
    className="mt-32"
  />
</div>
`;
